packages/sdk/python/src/latitude_sdk/reference.md

```
@@ -4471,6 +4471,350 @@ client.signals.list_traces(
+</dl>
+
+
+</dd>
+</dl>
+</details>
+
+<details><summary><code>client.signals.<a href="src/latitude_sdk/signals/client.py">unresolve</a>(...) -> SignalsLifecycleResponse</code></summary>
+<dl>
+<dd>
+
+#### 📝 Description
```

markdownlint-cli2 (0.23.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)